scripts/doc-checker.py

```diff
@@ -0,0 +1,123 @@
+"""
+Example usage:
+python scripts/doc-checker.py asv_bench/benchmarks/utils.py modin/pandas
+
+Notes
+-----
+    * the script can take some pathes to files or to directories
+"""
+
+import click
+import subprocess
+import os
+import ast
+
```

Comment:
instead of requiring script users to install `modin` locally I suggest doing something like this:
```suggestion
import sys
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

```